crates/iota-indexer/src/config.rs

```diff
@@ -103,6 +103,9 @@ pub struct JsonRpcConfig {
 
     #[clap(long)]
     pub rpc_client_url: String,
+
+    #[clap(long)]
+    pub grpc_client_url: Option<String>,
```

Comment:
Why is that part of the `JsonRpcConfig`? 
That sounds wrong to me. I think it should only be part of the `IngestionSources`, and I think the `JsonRpcConfig` is seen as the "old config" in the code?

Reply:
Yes it should be removed. Sorry I didn't take care of it correctly after rebasing. Fixed it at [f010fed](https://github.com/iotaledger/iota/pull/7511/commits/f010fed12482ddd9903589d7f276bbd9d5c52ea6).